packages/jest-validate/README.md

```diff
@@ -0,0 +1,92 @@
+# jest-validate
+
+Generic configuration validation tool that helps you with warnings, errors and deprecation messages as well as showing users examples of correct configuration.
+
+```
+npm install --save-dev jest-validate
+```
+
+## Usage
+
+```js
+import {validate} from 'jest-validate';
+
+validate(
+  config: Object,
+  validConfigSample: Object,
+  deprecatedConfig: ?Object,
+  options: ?Object
+);
```

Comment:
One more suggestion here, how about we do this:

```
validate(config, options);
```

where options is typed as:

```
ValidationOptions = {
  condition: (option: string, validOption: string): boolean,
  example?: Object,
  deprecatedConfig?: Object,
  deprecate: (config: Object, option: string): void,
  error: (option: string, configOption: string, validConfigOption: string, options: ValidationOptions): void // throws ValidationError,
  footer: string,
  titleDeprecation: string,
  titleError: string,
  titleWarning: string,
  unknown: (config: Object, option: string, options: ValidationOptions),
}
```

this seems like the simpler setup to me.